docs/contexts/index.md

```diff
@@ -41,7 +41,7 @@ A public key in JWK format. A JSON Web Key (JWK) is a JavaScript Object Notation
 }
 ```
 
-<h4 id="JwsVerificationKey2020"><a href="#JwsVerificationKey2020">JwsVerificationKey2020</a></h4>
+<h4 id="JsonWebKey2020"><a href="#JsonWebKey2020">JsonWebKey2020</a></h4>
```

Comment:
This is where the term is defined... the deprecated term, has no defintion now. can you handle this the same way you did in did core, define the older term and warn about using it?

Reply:
Sure, will get that fix in. Also is the table that registers these supposed to be generated by the code? If so, I'll update that as well so we actually register x25519 jwks.

Reply:
ahh, yes... that table does need to be updated. I suggest doing that in a separate PR

Reply:
To be super clear... I will comntinue to request changes, until I see the old term defintion preserved.

Reply:
Yup makes sense. Working on fixing them up today now that I've gotten a bit of time to get back to this.